.devops/ecommerce-deploy-pipelines.yml

```diff
@@ -0,0 +1,326 @@
+    dependsOn: [GreenDeployment_WaitForApproval, Get_Release_Version, Tag_Docker_Release]
+    condition: |
+      and(
+        eq(${{parameters.UAT_PROD_DEPLOY}}, true),
+        in(dependencies.Tag_Docker_Release.result, 'Succeeded', 'Skipped'),
+        in(dependencies.GreenDeployment_WaitForApproval.result, 'Succeeded', 'Skipped'),
+        or(
+          eq(variables['Build.SourceBranch'], 'refs/heads/main'),
+          startsWith(variables['Build.SourceBranch'], 'refs/tags')
+        )
+      )
+    variables:
+      app_version: $[ stageDependencies.Get_Release_Version.get_version.outputs['chart_current_version.appVersion'] ]
+    jobs:
+      - deployment: "Green_deployment"
+        displayName: "Green deployment"
+        pool:
+          name: pagopa-uat-linux
+        environment: "UAT"
+        strategy:
+          runOnce:
+            deploy:
+              steps:
+                # uninstall helm release used for blue instance
+                - task: HelmDeploy@0
+                  displayName: Un-install UAT blue version
+                  continueOnError: true
+                  inputs:
+                    kubernetesServiceEndpoint: $(UAT_KUBERNETES_SERVICE_CONN)
+                    namespace: ecommerce
+                    command: uninstall
+                    arguments: beta-$(K8S_IMAGE_REPOSITORY_NAME)
+                - checkout: self
+                  displayName: "Checkout"
+                - task: KubectlInstaller@0
+                - task: Bash@3
+                  name: update_chart_version
+                  displayName: "Setup helm microservice chart"
+                  inputs:
+                    targetType: "inline"
+                    script: |
+                      helm repo add microservice-chart https://pagopa.github.io/aks-microservice-chart-blueprint
+                      helm dep build helm
+                - template: azure-templates/helm-microservice-chart-deploy.yml
+                  parameters:
+                    DO_DEPLOY: true
+                    DO_BLUE_GREEN_DEPLOY: false
+                    ENV: "UAT"
+                    KUBERNETES_SERVICE_CONN: $(UAT_KUBERNETES_SERVICE_CONN)
+                    NAMESPACE: ecommerce
+                    APP_NAME: $(K8S_IMAGE_REPOSITORY_NAME)
+                    VALUE_FILE: "helm/values-uat.yaml"
+                    GREEN_VERSION: $(app_version)${{ variables.DOCKER_FILE_VERSION_SUFFIX }}
+                    #for jvm build add --values option targeting containing overrides from native helm values, such as required resources etc
+                    ${{ if eq(parameters['NATIVE_COMPILATION'], False) }}:
+                      ARGUMENTS: "--timeout 5m0s --values helm/valuesOverrides/overrides-jvm-uat.yaml --values helm/valuesOverrides/ecommerce/overrides-uat.yaml"
+                    ${{ else }}:
+                      ARGUMENTS: "--timeout 5m0s --values helm/valuesOverrides/ecommerce/overrides-uat.yaml"
+
+
+  # --- END Deploy UAT --- #
```

Comment:
why deploy pipeline stops to UAT env? please add also the prod steps with manual approval as per other services (see pagopa-checkout-auth-service) with prod values too

Reply:
it will be solved in this [PR](https://github.com/pagopa/pagopa-jwt-issuer-service/pull/7) to centralize deployment pipeline code using stages template